2-mandatory/3-stocks.js

```diff
@@ -48,7 +58,12 @@ function getAveragePrices(closingPricesForAllStocks) {
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
 function getPriceChanges(closingPricesForAllStocks) {
```

Comment:
This one looks good. One point here:
In this code - `stock[4] - stock[0]` - what would happen if the arrays we were passing in had more than 5 prices? Or less than 5 prices? Could you change this so it works for any number of prices?